pkg/apis/snapshots/v0alpha1/types.go

```diff
@@ -0,0 +1,50 @@
+package v0alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+type DashboardSnapshot struct {
+	metav1.TypeMeta `json:",inline"`
+	// +optional
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Info SnapshotInfo `json:"info"`
+
+	// The raw dashboard (??? openapi ???)
+	Dashbaord any `json:"dashboard"`
```

Comment:
```suggestion
	Dashboard any `json:"dashboard"`
```

:smile:

Reply:
🤕 i have never made a typo in my whole life